frontend/admin-client/src/app/bookings/booking-list/booking-list.component.ts

```diff
@@ -10,35 +12,110 @@ import {Booking} from '../../shared/models/booking';
 })
 export class BookingListComponent implements OnInit {
   bookings: Booking[] = null;
+  totalBookings: number;
+  pagedBookings: Booking[] = null;
+  pager: any = {};
   loading = false;
+  page = 1;
+  limit = 10;
   error = {isError: false, message: '', status: null};
 
-  constructor(private bookingService: BookingService) {
+  constructor(private bookingService: BookingService,
+              private  pagerService: PagerService,
+              private activatedRoute: ActivatedRoute, private router: Router) {
```

Comment:
The formatting here is off given the previously injected values are listed one per line

Reply:
oh yeah, I can definitely fit the first two into one line.